Source/cryptography/tests/cryptography_test/InterfaceTests.cpp

```diff
@@ -296,14 +326,51 @@ TEST(DH, Generate)
 
 }
 
-int main()
+int main(int argc, char **argv)
 {
     cg = WPEFramework::Cryptography::ICryptography::Instance("");
+    int len;
     if (cg != nullptr) {
-        vault = cg->Vault(cryptographyvault::CRYPTOGRAPHY_VAULT_NETFLIX);
+
+	for( int i= 1; i < argc; ++i )
+	{
+	    len = strlen(argv[i]);
+	    if ( (len == 9) && !strncmp( (const char*)argv[i], "--NETFLIX", len) )
```

Comment:
Could we have all the switches consistent in letter case?

Reply:
Could you elaborate on the comment. (NETFLIX, PLATFORM, DEFAULT) all are consistent in letter case. Are we talking about --testDefaultInstance , should it be changed to --TESTDEFAULTINTERFACE.

Reply:
Indeed. But I would make them all lowercase. Thanks.

Reply:
Updated. Please review. Thanks

Reply:
Looks good.